src/pages/iou/ReceiptSelector/NavigationAwareCamera.js

```diff
@@ -0,0 +1,51 @@
+import React, {useEffect, useState} from 'react';
+import {Camera} from 'react-native-vision-camera';
+import {useTabAnimation} from '@react-navigation/material-top-tabs';
+import {useNavigation} from '@react-navigation/native';
+import refPropTypes from '../../../components/refPropTypes';
+
+const propTypes = {
+    /* Forwarded ref */
+    forwardedRef: refPropTypes.isRequired,
+};
+
+// Wraps a camera that will only be active when the tab is focused or as asoon as it start to become focused.
+function NavigationAwareCamera(props) {
+    // Get navigation to get initial isFocused value (only needed once during init!)
+    const navigation = useNavigation();
+    const [isCameraActive, setIsCameraActive] = useState(navigation.isFocused);
+    // Get the animation value from the tab navigator. Its a value between 0 and the
+    // number of pages we render in the tab navigator. When we even just slightly start
+    // to scroll to the camera page, we want to activate the camera (so its as fast as possible active).
+    const tabPositionAnimation = useTabAnimation();
+
+    useEffect(() => {
+        const listenerId = tabPositionAnimation.addListener(({value}) => {
+            setIsCameraActive(value > 0 && value < 2);
```

Comment:
No, this would be incorrect. The animation value updates as soon as the user starts to swipe. As soon as the user starts to swipe to the camera [or clicks the header tab] the value updates to e.g. 0.0312 (animation running). And as soon as this happens we want to activate the camera.

Reply:
Maybe we should change the comments to make this more apparent?

Reply:
I updated the comments!